dashboards-observability/.cypress/integration/VisualizationCharts.spec.js/11_horizontalBar_chart.spec.js

```diff
@@ -0,0 +1,244 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/// <reference types="cypress" />
+import {
+    delay,
+    TEST_QUERIES,
+    querySearch,
+    landOnEventVisualizations,
+  } from '../../utils/event_constants';
+  
+  const numberOfWindow = 4;
+  const labelSize = 20;
+  const rotateLevel = 45;
+  const groupWidth = 10;
+  const groupWidthUpdated = 0.8;
+  const barWidth = 10;
+  const barWidthUpdated = 80;
+  const lineWidth = 7;
+  const lineWidthUpdated = 8;
+  const fillOpacity = 10;
+  const fillOpacityUpdated = 90;
+  const numberOfColor = 24;
+  
+  const renderHorizontalBarChart = () => {
+    landOnEventVisualizations();
+    querySearch(TEST_QUERIES[4].query, TEST_QUERIES[4].dateRangeDOM);
+    cy.get('[data-test-subj="configPane__vizTypeSelector"] [data-test-subj="comboBoxInput"]')
+      .type('Horizontal Bar')
+      .type('{enter}');
+  };
+  
+  describe('Render horizontal bar chart and verify default behaviour ', () => {
+    beforeEach(() => {
+      renderHorizontalBarChart();
+    });
+  
+    it('Render horizontal bar chart and verify by default the data gets render', () => {
+      cy.get('.xy').should('exist');
+    });
+  
+    it('Render horizontal bar chart and verify you see data configuration panel and chart panel', () => {
+      cy.get('.euiPanel.euiPanel--paddingSmall').should('have.length', numberOfWindow);
```

Comment:
are we able to add data-test-subj?

Reply:
this element has only class defined in it and no data-test-subj, so used the class only.